### PR TITLE
CAPT-1707 Add documentation for review apps on AKS

### PR DESCRIPTION
**Pull request by AbigailMcP**
## Changes in this PR

Add docs for accessing review apps.

## Guidance to review

[Check out the new markdown doc](https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/1d77c21a646da61b107f7d535205152c55f88e36/docs/connecting-to-azure.md).

### CODE REVIEW

File: docs/connecting-to-azure.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+3. Install kubetctl:
+
+   ```shell
+   brew install Azure/kubelogin/kubelogin
+   ```
+
+4. Then run:
+
+   ```shell
+   make review-aks get-cluster-credentials
```
Comment (asmega) [inline docs/connecting-to-azure.md:55]: i don't this command is right?
```
> make review-aks get-cluster-credentials
Makefile:28: *** Missing PR_NUMBER.  Stop.
```

Comment (AbigailMcP) [reply docs/connecting-to-azure.md]: ah thank you! updated :)

Comment (AbigailMcP) [reply docs/connecting-to-azure.md]: ah thank you! updated :)

**Review by slorek — APPROVED** [review]
Thanks, that's helpful. There are some our of date docs in the other files, could you remove/update these as appropriate either in this PR or a separate one? Thanks.

**Comment by AbigailMcP** [comment]
> Thanks, that's helpful. There are some out of date docs in the other files, could you remove/update these as appropriate either in this PR or a separate one? Thanks.

Thanks Steve, will tackle the out-of-date docs in a follow up PR.

**Pull request merged**